src/markup.c

```diff
@@ -44,6 +45,66 @@ static char *markup_br2nl(char *str)
         return str;
 }
 
+/*
+ * Remove hyperlinks of string without further reference
+ *
+ * (href information extraction should be done elsewhere)
+ *
+ */
+static char *markup_strip_a(char *str)
+{
+        str = string_replace_all("</a>", "</u></span>", str);
+
+        const char *start = str;
+        const char *end = NULL;
+
+        while ((start = strstr(start, "<a"))) {
+                end = strstr(start, ">");
+
+                // If we hit the end of the string, delete the trailing tag fully
+                unsigned int len = end ? (end-start)+1 : strlen(start);
+
+                str = string_replace_at(str, start-str, len, "<span color=\"blue\"><u>");
+
+                start++;
+        }
+
+        return str;
+}
+
+/*
+ * Remove image-tags of a string. If alt attribute given, use this as replacement.
+ */
+static char *markup_strip_img(char *str)
+{
+        const char *start = str;
+        const char *end = NULL;
+        const char *alt = NULL;
+        char *repl = NULL;
+
+        while ((start = strstr(start, "<img"))) {
+                end = strstr(start, ">");
+                alt = strstr(start, "alt=\""); // use alt=" as stated in the notification spec
```

Comment:
How about handling `src` as a URL too?

Reply:
Please, no. Wait for the feature request.